data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -487,6 +518,37 @@ spec:
                 description: Platform is configuration for machine pool specific to
                   the platform.
                 properties:
+                  alibabacloud:
+                    description: AlibabaCloud is the configuration used when installing
+                      on Alibaba Cloud.
+                    properties:
+                      instanceType:
+                        description: InstanceType defines the ECS instance type. eg.
+                          ecs.xn4.small
```

Comment:
This instance type is much too small for the default example. suggest: `ecs.g6.xlarge` or `ecs.g6.large`

Reply:
Ok